### ISSUE TITLE
[16.0] web_advanced_search: Add new value option with "in" does not work correctly

### ISSUE THREAD

**Issue by carlosdauden** [OCA/web#2965]
Only comment that the "add new value" option does not work correctly, I understand that this problem is different from the one solved by this PR and that functionally it is not important because the only way to use the "in" operator is using the debug mode.

But as I have detected it in my functional review, I leave it commented.

![image](https://github.com/user-attachments/assets/c6411731-9ec5-4414-8f6e-888ff455b977)

_Originally posted by @carlosdauden in https://github.com/OCA/web/pull/2964#pullrequestreview-2372014927_

**Comment by chienandalu** [comment]
Does it work in the core domain editor? (for example in a ir.filters)

**Comment by carlosdauden** [comment]
No, "in" operator is "not implemented" in domain editor for this version
